### ISSUE TITLE
Open file in new tab

### ISSUE THREAD

**Issue by ScriptorNY** [wincent/command-t#163]
Hello,

I've been trying lately to be able to open a file from the prompt in a new tab, but I was unable to.
The ctrl+t shortcut just opens the file in a new split. 

I tried to change the mapping related to the command:

```viml
let g:CommandTAcceptSelectionTabMap='<C-d>'
```

But the same thing happens.

Here are my command-t settings:

```viml
let g:CommandTMaxHeight=10


"
" buffer/split/tab re-use magic
"

function! s:BufHidden(buffer)
  let bufno = bufnr(a:buffer)
  let listed_buffers = ''

  redir => listed_buffers
  silent ls
  redir END

  for line in split(listed_buffers, "\n")
    let components = split(line)
    if components[0] == bufno
      return match(components[1], 'h') != -1
    endif
  endfor
  return 0
endfunction

function! s:GotoOrOpen(command, ...)
  for file in a:000
    " bufwinnr() doesn't see windows in other tabs, meaning we open them again
    " instead of switching to the other tab; but bufexists() sees hidden
    " buffers, and if we try to open one of those, we get an unwanted split.
    if bufwinnr(file) != -1 || (bufexists(file) && !s:BufHidden(file))
      exec "sb " . file
    else
      exec a:command . " " . file
    endif
  endfor
endfunction

command! -nargs=+ GotoOrOpen call s:GotoOrOpen(<f-args>)

let g:CommandTAcceptSelectionCommand = 'GotoOrOpen e'
let g:CommandTAcceptSelectionTabCommand = 'GotoOrOpen tabe'
let g:CommandTAcceptSelectionSplitCommand = 'GotoOrOpen sp'
let g:CommandTAcceptSelectionVSplitCommand = 'GotoOrOpen vs'
```

Thank you in advance for any help :)

**Comment by ScriptorNY** [comment]
So, I've done some research and it seems to be linked to the [vim-plugin](https://github.com/xolox/vim-session). If I'm not in an opened session everything works fine, whereas as soon as I save it as a new session and reopen the session, it keeps opening files in a new split.

Here are my vim-session settings if it can enlighten you:

```viml
set sessionoptions-=help
set sessionsoptions-=blank
```

Thanks again for any help :)

**Comment by wincent** [comment]
Good research tracking that down. I don't know anything about vim-plugin
though, other than what I saw from glancing at its README just now. I'd
suggest opening a ticket there and asking.

**Comment by ScriptorNY** [comment]
So, sorry for not answering you quicker :/ . I did some more digging and the problem does not seem to come from vim-session. I found out that sometimes, it just stops working, I can't open file as other than a split. So I tried to see if it maybe was a tabline problem, since I'm using airline, but I'm quite sure it isn't: I've disabled the plugin, and same result. It also doesn't seem to come from the fact I'm using gvim and not vim. 

Thank you for your support
